parity/src/test/java/org/web3j/protocol/parity/RequestTest.java

```diff
@@ -133,7 +133,16 @@ public void testParityKillAccount() throws Exception {
                 + "\"params\":[\"0x407d73d8a49eeb85d32cf465507dd71d507100c1\",\"hunter2\"],\"id\":1}");
         //CHECKSTYLE:ON
     }
-    
+
+    @Test
+    public void testParityListAccounts() throws Exception {
+        BigInteger maxQuantityReturned = BigInteger.valueOf(100);
+        web3j.parityListAccounts(maxQuantityReturned, null, null).send();
+
+        verifyResult("{\"jsonrpc\":\"2.0\",\"method\":\"parity_listAccounts\","
+                + "\"params\":[100,null,null],\"id\":1}");
```

Comment:
Please can the test case use non-null values. I'm happy to merge once updated with this.